freqtrade/exchange/exchange.py

```diff
@@ -1022,8 +1022,12 @@ def get_rate(self, pair: str, refresh: bool, side: str) -> float:
                 return rate
 
         conf_strategy = self._config.get(strat_name, {})
+        dont_use_order_book = (
+            ('use_order_book' in conf_strategy or side == "sell") and
+            conf_strategy.get('use_order_book', False)
+        )
 
-        if conf_strategy.get('use_order_book', False) and ('use_order_book' in conf_strategy):
+        if dont_use_order_book:
```

Comment:
The naming (`dont_use_order_book`) suggests the wrong thing (if `dont_use_order_book=True` - we'll be using orderbook prices) - and as such, is very confusing to even look at.